openassessment/xblock/static/js/src/lms/oa_staff_info.js

```diff
@@ -53,12 +53,52 @@ OpenAssessment.StaffInfoView.prototype = {
             function(html) {
                 // Load the HTML and install event handlers
                 $('#openassessment__student-info', view.element).replaceWith(html);
+                view.ScoreOverrideHandler();
             }
         ).fail(function(errMsg) {
                 view.showLoadError('student_info');
             });
     },
+    
+    /**
+     Upon request, submits an override grade.
+     **/
+    peerScoreOverride: function() {
+    	var sel = $('#openassessment__staff-info', this.element);
```

Comment:
What are we looking for here?

If we already have a reference to the element, can we just do this?
```javascript
var selector = $(this.element);
```